.gitignore

```diff
@@ -6,3 +6,5 @@ bin
 coverage
 pkg/*
 gemfiles/**/*.gemfile.lock
+.idea/
+.ruby-version
```

Comment:
These are specific to your environment and do not belong in the gem.

Reply:
Yeah, I'll remove that